src/main/java/ladder/controller/LadderGameController.java

```diff
@@ -0,0 +1,72 @@
+package ladder.controller;
+
+import java.util.List;
+import java.util.stream.IntStream;
+import ladder.domain.Height;
+import ladder.domain.Ladder;
+import ladder.domain.PlayerName;
+import ladder.domain.linegenerator.LinePatternGenerator;
+import ladder.domain.linegenerator.RandomBooleanSupplier;
+import ladder.dto.LadderDto;
+import ladder.dto.LineDto;
+import ladder.dto.PlayerNamesDto;
+import ladder.view.InputView;
+import ladder.view.OutputView;
+
+public class LadderGameController {
+    private final InputView inputView = new InputView();
+    private final OutputView outputView = new OutputView();
+
+    public void run() {
+        List<PlayerName> playerNames = inputPlayerNames();
+        Height height = inputHeight();
+
+        LinePatternGenerator lineGenerator = new LinePatternGenerator(new RandomBooleanSupplier());
```

Comment:
```suggestion
        LineGenerator lineGenerator = new LinePatternGenerator(new RandomBooleanSupplier());
```
이왕 인터페이스 만들어본거, 좀 더 유연한 방식을 사용해보죠!
지어주신 변수명과 딱 맞기도 하네요 :)

Reply:
넵넵 저걸 의도하고 한건데 단축키를 쓰다가 저렇게 되어버린것 같아요..ㅠㅠ